pkg/cloud/aws/client/client.go

```diff
@@ -36,11 +36,10 @@ type Interface interface {
 	DescribeSubnets(input *ec2.DescribeSubnetsInput) (*ec2.DescribeSubnetsOutput, error)
 	DescribeInstanceTypeOfferings(input *ec2.DescribeInstanceTypeOfferingsInput) (*ec2.DescribeInstanceTypeOfferingsOutput, error)
 
+	DeleteSecurityGroup(input *ec2.DeleteSecurityGroupInput) (*ec2.DeleteSecurityGroupOutput, error)
```

Comment:
This client interface is privately duplicated in **cloud-prepare** as `awsClient`. Do we plan to reconcile this? It seems at least the interface could be publicly defined in **cloud-prepare** and perhaps the implementation?

Reply:
The goal is to remove the interface in the addon.